Nginx_NodeJS_20_2023_Supervisord/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+IMAGE="nginx-nodejs-20-2023-supervisord"
+
+docker build -t $IMAGE .
+
+if [ "$SKIP_GOSS" = "true" ]; then
+    echo "Skipping dgoss tests"
+else
+  GOSS_SLEEP=15 dgoss run $IMAGE
```

Comment:
See `amzn2023-java17-nginx-supervisord` and `goss_wait.yml` for a better approach than `GOSS_SLEEP` - we need to make the other images use this too but if we're adding a new one let's use the new approach